README.md

```diff
@@ -40,11 +40,35 @@ by default `reqres_rspec` is not active (this may be configured!). To activate i
 
 Documentation will be put into your application's `/doc` folder
 
-## Upload to S3
+## Upload to Amazon S3
 
-By default ReqRes will use `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, `AWS_REGION` and `AWS_REQRES_BUCKET` environment variables. But you can alter that in configuration, see below.
+Set up following environment variables
 
-`REQRES_UPLOAD=1 REQRES_RSPEC=1 bundle exec rspec --order=defined`
+```
+AWS_ACCESS_KEY_ID
+AWS_SECRET_ACCESS_KEY
+AWS_REQRES_BUCKET
+```
+
+Then run
+
+`REQRES_UPLOAD=1 REQRES_RSPEC=AmazonS3 bundle exec rspec --order=defined`
```

Comment:
yes, i plan to rewrite readme anyway. it's not very organised and it must be split to smaller files